source/operations/install-deploy-manage/decommission-server-pool.rst

```diff
@@ -18,6 +18,11 @@ the deployment. MinIO automatically migrates data from the decommissioned pool
 to the remaining pools in the deployment based on the ratio of free space
 available in each pool.
 
+When upgrading an older set of hardware, add the new hardware before decommissioning the old hardware.
```

Comment:
```suggestion
For hardware replacement cycles where you replace old pool hardware with a new pool, you should add the new pool through expansion before starting the decommissioning of the old pool.
You must complete the *entire* expansion procedure before starting decommissioning.
Do not perform expand + decommission configuration changes in a single step.
```

Some tweaks to make this super, super explicit as to what not to do.

Reply:
I actually wonder if we can take this chunk into an H2 with a title like "Server Pool Replacement Procedure" and have htis guidance there.

Reply:
Moving it to an H3 under Considerations.

Reply:
And updating with your suggested phrasing.